### PR TITLE
New Unit Tests cases (car managment)

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/integration/pricing/export/convergent-charging/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { default } from '../../convergent-charging';
+export { default } from '../../dummy/DummyPricingIntegration';
```
Comment (LucasBrazi06) [inline src/integration/pricing/export/convergent-charging/index.ts:1]: This file should not be pushed.

File: src/integration/refund/export/concur/index.ts [hunk 2]
```diff
@@ -1 +1 @@
-export { default } from '../../concur';
+export { default } from '../../dummy/DummyRefundIntegration';
```
Comment (LucasBrazi06) [inline src/integration/refund/export/concur/index.ts:1]: This file should not be pushed.

File: src/integration/pricing/export/convergent-charging/index.ts (hunk 1 above)
Comment (mahdibenromdhane) [reply src/integration/pricing/export/convergent-charging/index.ts]: Done

File: src/integration/refund/export/concur/index.ts (hunk 2 above)
Comment (mahdibenromdhane) [reply src/integration/refund/export/concur/index.ts]: Done

**Comment by mahdibenromdhane** [comment]
> Unit tests are failing due maybe to my last changes and I wanted to check the logs but no where in the test execution I can find the tenant:
> 
> image
> 
> Like OCPP:
> image
> 
> Same remark as before, check how it works/has been done in similar areas and reproduce it.

Done :)

**Comment by LucasBrazi06** [comment]
> Done :)

Thanks a lot!

**Pull request merged**